practice-1/main.cpp

```diff
@@ -14,28 +15,19 @@ class Music {
     const string artist_name;
     unsigned int year;
 
-    Music() {
-        music_id = "";
-        artist_name = "";
-        year = 0;
-    };
```

Comment:
CLion brings up an error when assigning `music_id = ""` (and the other ones too).